src/ralph/export_to_ng/resources.py

```diff
@@ -133,6 +135,13 @@ def dehydrate_imei(self, asset):
             imei = ''
         return imei
 
+    def dehydrate_office_infrastructure(self, asset):
+        try:
+            service = asset.service_id
```

Comment:
first of all this field is named `service_name` in ralph assets. Second of all, `AttributeError` is not possible here (it will be if you'll call `asset.service.id`) - `asset.service_id` will be either number (>0) or `None`, so my suggestion is `service = asset.service_id or ''`, without any try-except